test/unit/index.js

```diff
@@ -5,7 +5,8 @@ var chai = require("chai"),
     sinon = require("sinon"),
     mocha = require("mocha");
 
-var Buttercup = require("../../source/module.js");
+var Buttercup = require("../../source/module.js"),
```

Comment:
Yes of course :) Still whole thing still needs an update for es6. I just added iocane so didn't worry about changing the declaration. I don't think `var` is necessarily bad but yes `const` would be nicer for consistency.